src/lib/CollectStore.js

```diff
@@ -19,90 +23,88 @@ const normalize = (dbObject, doctype) => {
 }
 
 export default class CollectStore {
-  constructor(context, options = {}) {
+  constructor(context, client, options = {}) {
+    this.client = client
     this.listener = null
     this.options = options
 
     this.categories = require('../config/categories')
     this.banksUrl = null
 
+    this.updateUnfinishedJob = this.updateUnfinishedJob.bind(this)
+    this.onAccountCreated = this.onAccountCreated.bind(this)
+    this.onAccountUpdated = this.onAccountUpdated.bind(this)
+    this.onAccountDeleted = this.onAccountDeleted.bind(this)
+    this.onTriggerCreated = this.onTriggerCreated.bind(this)
+    this.onTriggerDeleted = this.onTriggerDeleted.bind(this)
+
     this.initializeRealtime()
   }
 
   async initializeRealtime() {
-    jobs
-      .subscribeAll(cozy.client)
-      .then(subscription => {
-        subscription
-          .onCreate(job => this.updateUnfinishedJob(job))
-          .onUpdate(job => this.updateUnfinishedJob(job))
-      })
-      .catch(error => {
-        // eslint-disable-next-line no-console
-        console.warn(`Cannot initialize realtime for jobs: ${error.message}`)
-      })
+    this.realtime = new CozyRealtime({ cozyClient: this.client })
```

Comment:
I think there is a warning when you initialize cozyRealtime like this, prefer to use `new CozyRealtime({ client: this.client })`.

Reply:
See https://github.com/cozy/cozy-libs/blob/9e7707f9b83f086c3fae1841420af8c8adc6aebc/packages/cozy-realtime/src/index.js#L133

Reply:
Thanks !